### ISSUE TITLE
Error while installing with npm: "rollbackFailedOptional: verb npm-session 4069d2b65f4705a5"

### ISSUE THREAD

**Issue by darshantejani007** [paytm/paytm-mini-programs-nodejs-sdk#1]
As the first step says in the README.md, we have added the following dependency into our `package.json` as following

> "dependencies": {
        "paytm-mini-programs-nodejs-sdk": "https://github.com/paytm/paytm-mini-programs-nodejs-sdk.git",
        "@babel/runtime": "^7.7.7",
        "aes-js": "^3.1.2",     
        //our other dependecies. 
}

The second step in the documentation is to run the following command in terminal: 
> npm install paytm-mini-programs-nodejs-sdk

We're getting error in this very step. I'm attaching the error as the error-screenshot.png to this thread below. 

<img width="690" alt="error-screenshot" src="https://user-images.githubusercontent.com/6480543/83476927-8ec92380-a4af-11ea-8206-d68e71055a7f.png">

Describing the same error in the text, it says rollbackFailedOptional. Googling this error, I could not find any solution that worked for us. I also tried installing other npm packages, (e.g. aes-js) and they were installed without any errors. So the error occurs particularly only in the PayTM's package.

Here's our system info for further details:
OS: Mac OSX 10.15.2
Node Version: v13.12.0

We're in the final stages of integrating our Mini App PWA with PayTM, so can anyone help on how to proceed? Thanks for the same.

**Comment by shivamshekhar** [comment]
Hi @darshantejani007 ,

This issue was occurring due to missing .jshintignore and .jshintrc files, as these files specify the linting configuration for jshint module, i.e which files need to be excluded for linting purpose, etc.

Since, we have included a prepare script in our package.json, which checks linting before running npm install, for that particular module, the linting was failing due to the above mentioned missing files. We have included those files, so it should be working fine now.
